pkg/router/f5/types.go

```diff
@@ -120,6 +123,8 @@ type f5Policy struct {
 	// connections when more than one rule matches.  Typically we use best-match;
 	// other possible values are all-match and first-match.
 	Strategy string `json:"strategy"`
+
+	Legacy bool `json:"legacy"`
```

Comment:
I was curious on this one - thanks for explaining it. So if we ever decide to lock down to a specific version of the `F5` api, then we would have to remove this - maybe adding a comment here would be helpful. Thx